src/presentational-components/shared/breadcrubms.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+import { Breadcrumb, BreadcrumbItem } from '@patternfly/react-core';
+
+const RbacBreadcrumbs = (breadcrumbs) => {
+  return (
+    breadcrumbs ?  <Breadcrumb style={ { minHeight: 18 } }>
```

Comment:
You should have a default prop for `breadcrumbs` of an empty array. That way you can get rid of the if statement and return empty array instead of `null`.

Reply:
no if statement - using && instead